runtime/runtime-params-estimator/src/cases.rs

```diff
@@ -1,6 +1,4 @@
 use num_rational::Ratio;
-#[cfg(feature = "protocol_feature_evm")]
```

Comment:
Can we remove the feature from Cargo.toml as well?

Reply:
The test failure signals that we probably can't :( 

```


[2021-06-23T10:58:26Z] error[E0063]: missing field `evm_chain_id` in initializer of `ApplyState`
--
  | [2021-06-23T10:58:26Z]   --> runtime/runtime-params-estimator/src/testbed.rs:80:27
  | [2021-06-23T10:58:26Z]    \|
  | [2021-06-23T10:58:26Z] 80 \|         let apply_state = ApplyState {
  | [2021-06-23T10:58:26Z]    \|                           ^^^^^^^^^^ missing `evm_chain_id`
  | [2021-06-23T10:58:26Z]
  | [2021-06-23T10:58:26Z] error: aborting due to previous error
  | [2021-06-23T10:58:26Z]
  | [2021-06-23T10:58:26Z] For more information about this error, try `rustc --explain E0063`.
  | [2021-06-23T10:58:26Z] error: could not compile `runtime-params-estimator`
  | [2021-06-23T10:58:26Z]
  | [2021-06-23T10:58:26Z] To learn more, run the command again with --verbose.
  | [2021-06-23T10:58:26Z] warning: build failed, waiting for other jobs to finish...
  | [2021-06-23T10:58:27Z] error[E0063]: missing field `evm_chain_id` in initializer of `ApplyState`
  | [2021-06-23T10:58:27Z]   --> runtime/runtime-params-estimator/src/testbed.rs:80:27
  | [2021-06-23T10:58:27Z]    \|
  | [2021-06-23T10:58:27Z] 80 \|         let apply_state = ApplyState {
  | [2021-06-23T10:58:27Z]    \|                           ^^^^^^^^^^ missing `evm_chain_id`
  | [2021-06-23T10:58:27Z]
  | [2021-06-23T10:58:27Z] error: aborting due to previous error
  | [2021-06-23T10:58:27Z]
  | [2021-06-23T10:58:27Z] For more information about this error, try `rustc --explain E0063`.
```

https://buildkite.com/nearprotocol/nearcore/builds/1676#0567b2b9-073b-473d-a91c-aa58b588db35/85-691

Reply:
Yeah, nightly tests fail :(. Restored.